README.md

```diff
@@ -63,7 +63,7 @@ directory. The most logical place to start are the [docs for the `HasEncryptedAt
 
 ## Requirements
 
-* Laravel: 5.5, 5.6, 5.7, or 5.8
+* Laravel: 5.5, 5.6, 5.7, 5.8, or 6.x
 * PHP: 7.1, 7.2, or 7.3
```

Comment:
Since 7.1 is [already deprecated](https://www.php.net/supported-versions.php) and 7.2 is in security-only, maybe also look into PHP 7.4 support (add `allow-failure` CI tests for example)

Reply:
Yes, That's good idea. For now I was just looking into my own case where i needed Laravel 6 and php 7.3 support.

Reply:
I think that should be a separate PR. There will probably be some issues.